appendix/stata-guide.tex

```diff
@@ -153,151 +198,241 @@ \subsection{Abbreviating commands}
+often \texttt{C:/} on a PC or \texttt{/Users/} on a Mac.
+This ensures that you always get the correct file in the correct folder.
+\textbf{Do not use \texttt{cd} unless there is a function that \textit{requires} it.}
+When using \texttt{cd}, it is easy to overwrite a file in another project folder.
+Many Stata functions use \texttt{cd} and therefore the current directory may change without warning.
+Relative file paths are common in many other programming languages,
+but there they are always relative to the location of the file running the code.
 Stata does not provide this functionality.
 
-\textbf{Dynamic file paths} use globals that are set in a central master do-file to dynamically build your file
-paths. This has the same function in practice as setting \texttt{cd}, as all new users should only have to change these
-file path globals in one location. But dynamic absolute file paths are a better practice since if the
-global names are set uniquely there is no risk that files are saved in the incorrect project folder, and
-you can create multiple folder globals instead of just one location as with \texttt{cd}.
+\textbf{Dynamic} file paths use global macros for the location of the root folder.
+These globals should be set in a central master do-file.
+This makes it possible to write file paths that work very similarly to relative paths.
+It also achieves the functionality that setting \texttt{cd} is often intended to:
+executing the code on a new system only requires updating file path globals in one location.
+If global names are unique, there is no risk that files are saved in the incorrect project folder.
+You can create multiple folder globals as needed and this is encouraged.
 
 \codeexample{stata-filepaths.do}{./code/stata-filepaths.do}
 
-\subsection{Placing line breaks}
+\subsection{Line breaks}
 
 Long lines of code are difficult to read if you have to scroll left and right to see the full line of code.
-When your line of code is wider than text on a regular paper you should introduce a line break.
-A common line breaking length is around 80 characters, and Stata and other code editors
-provide a visible ``guide line'' to tell you when you should start a new line using \texttt{///}.
-This breaks the line in the code editor while telling Stata that the same line of code continues on the next row in the
-code editor. Using the \texttt{\#delimit} command is only intended for advanced programming
-and is discouraged for analytical code in an article in Stata's official journal.\cite{cox2005styleguide}
-These do not need to be horizontally aligned in code unless they have comments,
+When your line of code is wider than text on a regular paper, you should introduce a line break.
+A common line breaking length is around 80 characters.
+Stata and other code editors provide a visible ``guide line''.
+Around that length, start a new line using \texttt{///}.
+You can and should write comments after \texttt{///} just as with \texttt{//}.
```

Comment:
```suggestion
You can write comments after \texttt{///} just as with \texttt{//}, and that is usually a good thing.
```
Seemed like a too strong recommendation with `should` as there is many times there is nothing to write.